packages/components/src/components/carousel/carousel.examples.md

```diff
@@ -240,6 +273,15 @@ export default class Code extends Vue {
   ${this.getSlides(3)}
 </p-carousel>`;
 
+  headingSize: CarouselHeadingSize = 'x-large';
+  headingSizes = CAROUSEL_HEADING_SIZES;
+  get headingMarkup() {
```

Comment:
Don't we usually call this `headingSizeMarkup` so the relation is clear to the model?